backend/app/models/public_schools_analysis.rb

```diff
@@ -261,9 +268,9 @@ def school_object(school, level)
       district: school[:district],
       subdistrict: school[:subdistrict],
       source: school[:source],
-      capacity: school["#{db_level}_capacity".to_sym],
-      capacityFuture: school["#{db_level}_capacity".to_sym],
-      enroll: school["#{db_level}_enroll".to_sym],
+      capacity: school["#{db_level}c".to_sym],
+      capacityFuture: school["#{db_level}c".to_sym],
```

Comment:
What's the difference between `capacity` and `capacityFuture`?

Reply:
Had the same question -- maybe I'll make a todo or note here for future investigation